taker-frontend/src/components/DollarAmount.tsx

```diff
@@ -0,0 +1,28 @@
+import { Text, useColorModeValue } from "@chakra-ui/react";
+import * as React from "react";
+
+interface DollarAmountProps {
+    amount: number;
+    maximumFractionDigits?: number;
+    textColor?: {
+        light: string;
+        dark: string;
+    };
+}
+
+export default function DollarAmount({ amount, textColor, maximumFractionDigits }: DollarAmountProps) {
+    const price = Math.floor(amount * 100.0) / 100.0;
+
+    const colorMode = useColorModeValue(textColor?.light, textColor?.dark);
+    // Note : `undefined` will make use of the system local. In our case, the user's browser settings.
+    let formater = Intl.NumberFormat(undefined, {
```

Comment:
Nit:

```suggestion
    let formatter = Intl.NumberFormat(undefined, {
```

Note: There is actually a react package for these intl APIs: https://formatjs.io/docs/getting-started/installation/

Reply:
fixed the typo. 
I'd say we use the package once we really want to go for internationalization